Lesson2/src/main/java/pl/solvd/university/administration/AdmissionsOffice.java

```diff
@@ -0,0 +1,147 @@
+package pl.solvd.university.administration;
+
+import pl.solvd.university.departments.Faculty;
+import pl.solvd.university.departments.Speciality;
+import pl.solvd.university.documents.Pass;
+import pl.solvd.university.people.Applicant;
+import pl.solvd.university.state.PassExams;
+import pl.solvd.university.state.Registration;
+import pl.solvd.university.state.SaveLoadFiles;
+
+import java.io.IOException;
+import java.util.*;
+
+import static pl.solvd.university.Main.*;
+
+public class AdmissionsOffice {
+    public static List<Applicant> applicants = new ArrayList<>();
+
+    public static void chooseSpeciality() throws IOException, ClassNotFoundException {
+        SaveLoadFiles.load("Lesson2/src/main/resources/state.bin");
+        System.out.println("\nHere you can see the Faculties of the University: ");
+        LOG.info("The list of Faculties is displayed");
+        for (Faculty categories : Faculty.values()) {
+            System.out.println(categories);
+        }
+        System.out.println("\nHere you can see the Specialities of the University: ");
+        LOG.info("The list of Specialities is displayed");
+        for (Speciality type : Speciality.values()) {
+            System.out.println(type);
+        }
+        System.out.println("\nWrite what specialty you want to study:\n1 - CONCERT_PERFORMER,\n2 - VOCAL_SINGER,\n3 - MUSICAL_DIRECTOR,\n4 - CHOREOGRAPHER\n");
```

Comment:
You should not hardcode this, this message should be automatically build, depending on available specialities

Reply:
Done